### PR TITLE
Fix grammar issues and warning

### PR DESCRIPTION
**Pull request by Rajdave69**
## Summary

This PR fixes a lot of spelling and grammar issues within multiple files. And it also fixes very minor warnings within the code

## Checklist

- [ ] This PR has breaking changes
- [ ] This PR fixes an issue
- [x] This is **not** a code change
- [x] If code changes were made they have been tested
- [x] I have properly changed the docs to appeal to this change

### CODE REVIEW

**Comment by VincentRPS** [comment]
@Rajdave69 please rebase the PR so I can merge it

**Comment by Rajdave69** [comment]
oops

**Comment by Rajdave69** [comment]
I think I've rebased it.

File: examples/interactions/commands/application_commands.py [hunk 1]
```diff
@@ -26,7 +27,7 @@ async def favorite(
     inter,
     anime: pycord.Option = pycord.Option(
         # The type of input the user will put,
-        # for this example it's integer to support autocomplete.
+        # for this example its integer to support autocomplete.
```
Comment (Dorukyum) [inline examples/interactions/commands/application_commands.py:30]: This is supposed to mean "it is", hence "it's"
```suggestion
        # for this example it's integer to support autocomplete.
```

**Review by Dorukyum — DISMISSED** [review]

File: examples/interactions/commands/application_commands.py (hunk 1 above)
Comment (Rajdave69) [reply examples/interactions/commands/application_commands.py]: Fixed

**Comment by Rajdave69** [comment]
fixed

**Review by baronkobama — APPROVED** [review]
LGTM

**Review by VincentRPS — APPROVED** [review]

**Pull request merged**